code/modules/chemistry/Chemistry-Reagents.dm

```diff
@@ -333,6 +333,24 @@ datum
 					holder.remove_reagent(reagent_id, (amount * holder.reagent_list[reagent_id].flushing_multiplier))
 			return
 
+        ///Numb one or more limbs without stacking status effects or targeting limbs that don't exist
+		proc/numb_limb(mob/M, duration, count=1, random=TRUE, list/target_limbs=list("l_arm", "l_leg", "r_arm", "r_leg"),)
+			if(!ishuman(M))
+				return
+			var/mob/living/carbon/human/H = M
+			if(!H.limbs)
+				return // no limb datum
+			var/list/obj/item/parts/limbs_to_numb = list()
+			for(var/target in target_limbs)
+				var/obj/item/parts/limb = H.limbs.vars[target]
+				if(istype(limb))
+					limbs_to_numb += limb
+			if(!length(limbs_to_numb))
+				return // no limbs
+			if(random)
+				shuffle_list(limbs_to_numb)
+			for (var/i in 1 to min(count, length(limbs_to_numb)))
+				M.setStatusMin("numb_[limbs_to_numb[i].slot]", duration)
```

Comment:
why is this a proc on reagent

Reply:
It allows for customizability and ease of application.
A good example would be a set choice of non-random limb stuns, stunning 2 different limbs at once or setting only arms or legs for stunning, without having to apply all the logic in each possible instance in the future.

Reply:
But why is this on reagent and not on human?

Reply:
Oh, wow i don't think i realized that. Frankly it's been so long that i made this proc that i don't quite recall my reason for putting it on reagent. I can definitely change it though.

Reply:
Moved to human.dm